build.env

```diff
@@ -9,7 +9,7 @@
 # get proporly expanded.
 #
 # cephcsi image version
-CSI_IMAGE_VERSION=v3.12-canary
+CSI_IMAGE_VERSION=v3.12.1
 
 # cephcsi upgrade version
 CSI_UPGRADE_VERSION=v3.11.0
```

Comment:
we are not updating `CSI_UPGRADE_VERSION` to lates?

Reply:
That needs to be done in devel branch, not release branch.

Reply:
https://github.com/ceph/ceph-csi/pull/4785